Docker/README.md

```diff
@@ -48,64 +48,82 @@ To run analysis __locally__:
 
    where `source-directory` and `output-directory` are full local paths to, respectively, the project source code directory and the analysis results directory.
 
-   This command will run the analysis on your source code and start the web server to provide a convenient view of the results. Open [`http://localhost:8080`](http://localhost:8080) in your browser to examine the found problems and performed checks. Here, you can also reconfigure the analysis. See the [UI section](../UI/README.md) of this guide for details.
+   This command will run the analysis on your source code and start the web server to provide a convenient view of the results. Open [`http://localhost:8080`](http://localhost:8080) in your browser to examine the found problems and performed checks. Here, you can also reconfigure the analysis. See the [UI section](../UI/README.md) of this guide for details. Press `Ctrl-C` in docker console when you are done, to stop the web server.
```

Comment:
Let's mb go the other way around? "When done, you can stop the web server by pressing `Ctrl-C` in the Docker console."